src/www/model/server.ts

```diff
@@ -43,12 +43,18 @@ export interface Server {
 
   // Checks whether the server is reachable.
   checkReachable(): Promise<boolean>;
+
+  // Checks whether we can fix the underlying outline services for this server.
+  canFixServices(): Promise<boolean>;
+
+  // Try to fix the underlying outline services for this server.
+  tryFixServices(): Promise<void>;
```

Comment:
Let's call it `installService` or something like `{configure|setup}Vpn`.
No `try` in the method name.

Also, the scope for this is not really a Server.
The service is shared.

I think we need to think more about how we may want to design this, but for now:
- add an install/configure/setup method to OutlineServerRepository. That's the right scope, and it doesn't have to be in the model, as it's mostly an application-specific concern.
- Add a corresponding method to the Tunnel, which gets called by the repo, and provides the actual implementation.

Reply:
BTW, Android has the concept of "preparing" the VPN before using it. We may want to model it in a similar way as Electron in the future: https://developer.android.com/guide/topics/connectivity/vpn

Reply:
👍

Reply:
Sounds good

Reply:
I introduced a new `VpnInstaller` interface to do the work. So it decouples with `ServerRepository` or `Tunnel`.